utils/helper-utils.ts

```
@@ -0,0 +1,7 @@
+export function chunkArray(array: any[], chunkSize: number) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)